app/controllers/announcements_controller.rb

```diff
@@ -25,7 +25,7 @@ def edit
   def update
     @announcement = Announcement.find(params[:id])
     @coronavirus_page = @announcement.coronavirus_page
-    if @announcement.update(announcement_params)
+    if @announcement.update(announcement_params) && draft_updater.send
```

Comment:
Nice atomic commit 🤗

Reply:
🥇 :)

Reply:
Oh dear, your nice atomic commit got a bit broken in the rebase. Nevermind!

Reply:
oh damn it!!